pages/linux/pacman-upgrade.md

```diff
@@ -0,0 +1,28 @@
+# pacman --upgrade
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --upgrade --help`
+
+- Install one or more packages from files:
+
+`sudo pacman --upgrade {{path/to/package1}} {{path/to/package2}}`
```

Comment:
Do pacman packages have a file extension by convention? If so, it'd be nice to update these examples here to include it. Same goes for the other examples.

If not, just ignore this message.

Reply:
```suggestion
`sudo pacman --upgrade {{path/to/package1.pkg.tar.zst}} {{path/to/package2.pkg.tar.zst}}`
```